src/gei/Handlers/MigrateRepoCommandHandler.cs

```diff
@@ -242,7 +249,8 @@ private async Task<bool> DetermineIfBlobCredentialsRequired(MigrateRepoCommandAr
         if (args.GhesApiUrl.HasValue())
         {
             _log.LogInformation("Using GitHub Enterprise Server - verifying server version");
-            var ghesVersion = await _sourceGithubApi.GetEnterpriseServerVersion();
+            // var ghesVersion = await _sourceGithubApi.GetEnterpriseServerVersion();
+            var ghesVersion = "3.0.4";
```

Comment:
```suggestion
            var ghesVersion = await _sourceGithubApi.GetEnterpriseServerVersion();
```

Reply:
Addressed in https://github.com/github/gh-gei/pull/787/commits/2afeca3056b0bd970e90cee0a09daedb5ec0c9c5! 🎉